imap_processing/tests/lo/test_lo_science.py

```diff
@@ -115,6 +116,36 @@ def segmented_pkts_fake_data():
     return dataset
 
 
+@pytest.fixture()
+def fake_spin_data():
+    dataset = xr.Dataset(
+        data_vars=dict(
+            num_completed=(["epoch"], np.array([0, 0])),
+            acq_start_sec=(["epoch"], np.array([0, 0])),
+            acq_start_subsec=(["epoch"], np.array([0, 0])),
+            acq_end_sec=(["epoch"], np.array([0, 0])),
+            acq_end_subsec=(["epoch"], np.array([0, 0])),
+        ),
+        coords=dict(epoch=(["epoch"], np.array([0, 1]))),
+    )
+    spin_fields = [
+        "start_sec_spin",
+        "start_subsec_spin",
+        "esa_neg_dac_spin",
+        "esa_pos_dac_spin",
+        "valid_period_spin",
+        "valid_phase_spin",
+        "period_source_spin",
+    ]
+
+    for spin_field in spin_fields:
```

Comment:
If you could put `i` into the actual values rather than just 0s, you could test the ordering is done correctly.

Reply:
Yeah good point. I'll update this